src/system/startup.c

```diff
@@ -33,21 +32,24 @@ extern void invoke_install_hot_table();
 // gives the compiler instructions on the priority of the constructor,
 // from 0-~65k. The first 0-100 priorities are reserved for language
 // implementation.
-__attribute__((constructor(101))) static void pros_init(void) {
+__attribute__((constructor(101))) static void pros_init_a(void) {
```

Comment:
maybe `_1` and `_2` suffix instead of `_a` and `_b`? I was initially wondering what `_a` stood for.

Reply:
a more verbose `stage_1` and `stage_2` would do the trick too